src/web/mjs/engine/Settings.mjs

```diff
@@ -39,6 +39,9 @@ export default class Settings {
         } catch ( e ) {
             docs = '.';
         }
+        //HYPOFLEX
+        this.fs = require( 'fs' );
```

Comment:
Use `fs-extra` instead, it provides everything from `fs` but returns more convenient promises instead of callback functions

Reply:
the `fs` function i grabbed from the `storage` file, where `fs` is still used, maybe good to change it there to `fs-extra` as well?

Reply:
Yes, good idea